docs/docs/tools-and-integrations/integrations/keptn.md

```diff
@@ -0,0 +1,172 @@
+# Running Tracetest with Keptn
```

Comment:
This documentation is the same as presented on https://github.com/kubeshop/tracetest/tree/main/examples/keptn-integration (I only needed to fix the links to the files).

After the contents of this documentation are published on docs.tracetest.io, I'll open another PR changing the docs there to reference this document.